source/pdo_sqlsrv/config.w32

```diff
@@ -38,7 +38,12 @@ if( PHP_PDO_SQLSRV != "no" ) {
             ADD_FLAG( "CFLAGS_PDO_SQLSRV", "/D ZEND_WIN32_FORCE_INLINE" );
             if (VCVERS >= 1913) {
                 ADD_FLAG("CFLAGS_PDO_SQLSRV", "/Qspectre");
-            } 
+            } else if (VCVERS == 1900) {
+                var subver1900 = probe_binary(PHP_CL).substr(6);
+                if (subver1900 >= 24241) {
```

Comment:
I didn't see these version numbers in the link you provided - where are they listed?

Reply:
It is the compiler cl version after patching VS 2015 Update 3.